content/guides/composite_monitors.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Guide to Composite Monitors
+kind: guide
+listorder: 9
+---
+
+***If you're unfamiliar with the basics of Datadog Monitors, first read the [Guide to Monitors](/guides/monitors)***
+
+---
+
+Composite monitors let you combine many individual monitors into one so that you can define more specific alert conditions. You can choose up to 10 existing monitors - monitor A and monitor B, say - and then set a trigger condition using boolean operators (e.g. “A && B”). The composite monitor will trigger when its individual monitors' statuses simultaneously have values that cause the composite's trigger condition to be true.
+
+With regard to configuration, a composite monitor is independent of its constituent monitors. You can modify the notification policy of a composite monitor without affecting the policies of its constituent monitors, and vice versa. Furthermore, deleting a composite monitor will not delete its constituent monitors; a composite monitor does not own other monitors, it only uses them. Many composite monitors may reference the same individual monitor.
+
+_Note: this guide refers variously to 'individual monitors', 'constituent monitors', and 'non-composite monitors'. They all mean the same thing: the monitors that a composite monitor uses to calculate its status._
```

Comment:
As it's written now, this feels necessary to say. I'd like to avoid using three different terms to refer to the same thing, but looking across all usages, I can find appropriate (IMO) usages of each. I'm open to scrapping one, though, if others think three is too many.